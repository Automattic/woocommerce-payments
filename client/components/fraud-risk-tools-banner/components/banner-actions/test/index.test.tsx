/**
 * External dependencies
 */
import React from 'react';
import { render } from '@testing-library/react';

/**
 * Internal dependencies
 */
import BannerActions from '..';

const mockHandleDontShowAgainOnClick = jest.fn();

describe( 'BannerActions', () => {
	it( 'renders', () => {
		const { container: bannerActionsComponent } = render(
			<BannerActions
				handleDontShowAgainOnClick={ mockHandleDontShowAgainOnClick }
			/>
		);

		expect( bannerActionsComponent ).toMatchSnapshot();
	} );
} );
