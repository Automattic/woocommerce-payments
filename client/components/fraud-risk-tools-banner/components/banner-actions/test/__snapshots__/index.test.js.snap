// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerActions renders with dont show again button when remindMeCount greater than or equal to 3 1`] = `
<div>
  <div
    class="discoverability-card__actions"
  >
    <a
      class="components-button is-primary"
      href="/wp-admin/admin.php?page=wc-settings&tab=checkout&anchor=%23fp-settings&section=woocommerce_payments/"
    >
      Learn More
    </a>
    <button
      class="components-button is-tertiary"
      type="button"
    >
      Remind me later
    </button>
    <button
      class="components-button is-tertiary"
      type="button"
    >
      Don't show me this again
    </button>
  </div>
</div>
`;

exports[`BannerActions renders without dont show again button when remindMeCount less than 3 1`] = `
<div>
  <div
    class="discoverability-card__actions"
  >
    <a
      class="components-button is-primary"
      href="/wp-admin/admin.php?page=wc-settings&tab=checkout&anchor=%23fp-settings&section=woocommerce_payments/"
    >
      Learn More
    </a>
    <button
      class="components-button is-tertiary"
      type="button"
    >
      Remind me later
    </button>
  </div>
</div>
`;
